### PR TITLE
[DCB][SUPP-579] update Welsh translations

### PR DESCRIPTION
**Pull request by DarrenCB**

### CODE REVIEW

File: app/messages/Messages.scala [hunk 1]
```diff
@@ -352,6 +352,11 @@ object Messages {
       welsh   = "Dysgu sut i gofrestru"
     )
 
+    val `If you need to speak to an adviser call us on 0300 200 3835 at the Business Support Service to talk about your payment options.`: Message = Message(
```
Comment (barnesjake) [inline app/messages/Messages.scala:355]: Where's this new content come from? I can't see it in the prototype v3.1

File: app/messages/Messages.scala [hunk 2]
```diff
@@ -1318,9 +1323,9 @@ object Messages {
       welsh   = "Yr hyn sy’n digwydd nesaf"
     )
 
-    val `HMRC will send you a letter within 5 working days with your payment dates.`: Message = Message(
-      english = "HMRC will send you a letter within 5 working days with your payment dates.",
-      welsh   = "Bydd CThEF yn anfon llythyr atoch cyn pen 5 diwrnod gyda’ch dyddiadau talu."
+    val `We will send a letter by post confirming the set up of your Direct Debit instruction within 5 working days.`: Message = Message(
```
Comment (barnesjake) [inline app/messages/Messages.scala:1326]: Same with this

File: app/messages/Messages.scala (hunk 1 above)
Comment (DarrenCB) [reply app/messages/Messages.scala:355]: Second paragraph:
https://supp-prototype.herokuapp.com/3-1/unhappy/not-enrolled
https://supp-prototype.herokuapp.com/3-1/vat/not-enrolled

File: app/messages/Messages.scala (hunk 2 above)
Comment (DarrenCB) [reply app/messages/Messages.scala:1326]: Confirmation page prototype:
https://supp-prototype.herokuapp.com/3-1/confirmation

Karen just confirming with business if wording needs to be applied for both EPAYE & VAT. For now, updating. If need to make them separate, can do that later.

File: app/messages/Messages.scala (hunk 1 above)
Comment (barnesjake) [reply app/messages/Messages.scala:355]: I feel like it should be one message or the other, and maybe prototype isn't synced up - both messages are basically the same just different word ordering within the sentence. Also, can we get a ticket for this? I don't want to start merging in content changes without a ticket as it's technically not properly testable - we can't just say "check protoytype"

We need a ticket saying, it was "this" it should be "this" and list each page

Comment (DarrenCB) [reply app/messages/Messages.scala:355]: Also in Welsh spreadsheet: https://docs.google.com/spreadsheets/d/1CdBoO5ASk4iwsvHStPkT36tbIZNduLOC/edit#gid=47327513

Comment (DarrenCB) [reply app/messages/Messages.scala:355]: I will create a ticket & double-check exactly which wording with Rocco. But I am following the spreadsheet of content in ticket 579.

**Review by barnesjake — APPROVED** [review]

**Review by barnesjake — APPROVED** [review]

**Pull request merged**